stockflux-securities-master/src/enums.js

```diff
@@ -0,0 +1,13 @@
+export const tableEnum = {
```

Comment:
I think these two could be named a bit better. Maybe something like `TableState` and `InputFormState`. Putting type type `enum` as part of the name is generally not a good idea. Descriptive names generally read much better.

Reply:
I see what you mean. However my reason for the name was that `TableState ` is too similar to `tableState` which I'm currently using for the components state. Would tableStateConstants be a better name? Is there a convention for capitalisation?

Reply:
The intention with this "enum" (which isn't really an enum since they're not supported in JS) is to avoid mistakes and limit the set of values which are considered to be correct. In this instance we're almost using your enum as if it was a type (enums are an actual type in Java and I think TypeScript too which compiles to JS). With this in mind, I would treat the enum as if it was a type, or a class and in which case the convention would usually be to `StudlyCase`. Variables in JavaScript tend to be `camelCase`, so it gives quite a good way of distinguishing what is what.

I'd definitely go for `TableState` and `InputFormState` or similar. You should avoid trying to use the name of the type as part of the name of something. For example, if you were to create a class, you would never name it `SomethingClass` or if you created a variable called `state` that had a boolean type you wouldn't call `stateBoolean`. Names of things should be descriptive as possible but also quite concise.

In your case I would be inclined to rename your `tableState` variable (the variable, not the enum) to something like `state`, if that can be done. If not, then maybe something else. Naming things is renowned for being one of a software developers greatest challenges and it might take some time to get right but it's good to try to get into the mindset of why things are named how they are and pick up good habits early.

> There are only two hard things in Computer Science: cache invalidation and naming things.
> -- Phil Karlton

Reply:
Done this now